src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -99,13 +97,9 @@ const SignedChatScreen = () => {
           onThreeDotPress={goToChatInfoPage}
           avatar={selectedChannel?.channelPicture}
           type={SIGNED}
-          user={
-            selectedChannel?.rawJson?.channel?.anon_user_info_emoji_code
-              ? `Anonymous ${selectedChannel?.rawJson?.channel?.anon_user_info_emoji_name} `
-              : getChatName(selectedChannel?.name, profile?.myProfile?.username)
-          }
-          anon_user_info_emoji_code={selectedChannel?.rawJson?.channel?.anon_user_info_emoji_code}
-          anon_user_info_color_code={selectedChannel?.rawJson?.channel?.anon_user_info_color_code}
+          user={selectedChannel?.name}
+          anon_user_info_emoji_code={selectedChannel?.anon_user_info_emoji_code}
+          anon_user_info_color_code={selectedChannel?.anon_user_info_color_code}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in how the `user` prop is passed to the component seems to be a simplification, but it's important to ensure that this doesn't introduce any regressions. The old code had a conditional check for `selectedChannel?.rawJson?.channel?.anon_user_info_emoji_code` and used a different value (`getChatName(selectedChannel?.name, profile?.myProfile?.username)`) if it was not present. If the new code doesn't need to handle this case, then it's fine. Otherwise, you might want to reintroduce the conditional logic.
```diff
- user={selectedChannel?.name}
+ user={
+   selectedChannel?.anon_user_info_emoji_code
+     ? `Anonymous ${selectedChannel?.anon_user_info_emoji_name} `
+     : getChatName(selectedChannel?.name, profile?.myProfile?.username)
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->